mypy-strict.ini

```
@@ -40,6 +40,7 @@ files = tools/codegen/gen.py,
     tools/test_history.py,
     torch/testing/_internal/mypy_wrapper.py,
     torch/testing/_internal/print_test_stats.py,
+    torch/testing/_internal/framework_utils.py,
```

Yay :smiley: although perhaps consider keeping these in alphabetical order?
```suggestion
    torch/testing/_internal/framework_utils.py,
    torch/testing/_internal/mypy_wrapper.py,
    torch/testing/_internal/print_test_stats.py,
```